cmd/cosign/cli/verify/verify.go

```diff
@@ -138,22 +139,31 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 	co.SigVerifier = pubKey
 
 	for _, img := range images {
-		ref, err := name.ParseReference(img)
-		if err != nil {
-			return errors.Wrap(err, "parsing reference")
-		}
-		ref, err = sign.GetAttachedImageRef(ref, c.Attachment, ociremoteOpts...)
-		if err != nil {
-			return errors.Wrapf(err, "resolving attachment type %s for image %s", c.Attachment, img)
-		}
+		if c.LocalImage {
+			verified, bundleVerified, err := cosign.VerifyLocalImageSignatures(ctx, img, co)
```

Comment:
A single directory can contain multiple images -- I'd consider making `LocalImage` a string that points to the path instead of a bool and use `img` to address the image within that path.

The UX here gets a little funky -- for a directory with a single image, you probably want to just default to it and not require users to pass it as an arg as well.

@priyawadhwa might have some ideas here.

Reply:
i think we [overwrite](https://github.com/sigstore/cosign/blob/main/pkg/oci/layout/write.go#L29) the directory every time we run `cosign save` so only a single image should be allowed at the moment -- eventually if we add support for multiple images in a directory then we'll need to figure out what the UX would look like

Reply:
Thanks for pointing this out! I agree that since we don't support persisting multiple images in a single directory, this shouldn't be an issue for now. 

For the future, I like the approach of specifying the image along with the path by flag.